### DISCUSSION TITLE
Привет всем, прошу начать активную разработку вашей удобной площадки!

### DISCUSSION THREAD

**Discussion by AsakiQ** [stoatchat/discussions#800] [General]
Привет всем, прошу начать активную разработку вашей удобной площадки! Желаю искренне разработчикам счастья и здоровья. 
Пожалуйста добавте кастомные ссылки как у дискорда на сервера, в виде постоянных. Пример прямых ссылок: https://rvlt.gg/revolt . Так же пожалуйста добавте как можно скорее, я хочу пользоваться ТОЛЬКО ВАШЕЙ площадкой, не Guilded и TeamSpeak. Просто чуть доработать как можно быстрее и 2 больших сервера с площадки Discord перейдут дружно к вам. К сожалению с политикой моей страны сейчас данные сервера будут активно умирать, я же в свою очередь хотел бы перегнать трафик к вам.
![image](https://github.com/user-attachments/assets/8f11c8c3-0866-4a2c-84b3-3a48d54a8e41)

**Comment by AsakiQ** [comment]
Гайс оставьте пожалуйста коммент и продвиньте моё предложение! Это поможет нам создать комьюнити!


Guys please leave a comment and repost my post! This will help us create a community!

**Comment by baccaraaa** [comment]
Это ваш сервер? Согласен, нужно добавить немного базовых функций и это отличное импортозамещение

**Reply by AsakiQ** [reply]
Да это оба моих сервера, к сожалению на других площадках владельцы забрали определённые ссылки, и многие из них делали грязные вещи по типу педофилии, скама и так далее, я же хочу собрать вокруг себя комьюнити которое будет максимально адекватно общаться, и отдельные места для токсиков. Думаю Revolt сможет помочь в реализации моей идеи.

**Comment by JoCat** [comment]
> прошу начать активную разработку

Думаю они и без ваших указаний знают что им делать))

Лучше сформировать это как конкретные задачи с хорошим описанием 😉
Это будет лучше в данной ситуации, чём что-то от них требовать в таком тоне.

**Reply by AsakiQ** [reply]
Это не требовательный тон, просто если я буду спать а эту фичу добавят, я не переведу комьюнити на чужой сервер и часть просто потеряется в том же Тимспике.

**Reply by kostya-main** [reply]
Если вам так нужны эти функции возьмите найдите разработчика (или сами) напишите данный функционал и отправите запрос на слияние (PR). Они явно сейчас будут не функционал пилить а развиваться серверную инфоструктуру.

**Reply by JoCat** [reply]
> Это не требовательный тон, просто если я буду спать а эту фичу добавят, я не переведу комьюнити на чужой сервер и часть просто потеряется в том же Тимспике.

Ну что могу сказать, следите за проектом, и будет вам счастье

**Comment by minemokinesan** [comment]
Добрый день
поддерживаю тему и прошу ввести 2 конкретных изменения площадки:
-push-to-talk в голосовых каналах
-прямая ссылка на сервер
p. s. Русское комьюнити будет крайне благодарно за предоставление аналога дискорда в данной ситуации, надеюсь вы оправдаете наши ожидания

Good day
I support the topic and ask for 2 specific changes to the platform:
-push-to-talk in voice channels
-direct link to the server
p.s. The Russian community will be extremely grateful for providing an analogue of Discord in this situation, I hope you will live up to our expectations
